ansible/roles/ocp-workload-integreatly/tasks/workload.yml

```diff
@@ -6,10 +6,21 @@
     self_signed_certs_enabled: False
   when: lets_encrypt_production|bool
 
+- name: Generate evals user password
+  set_fact:
+    evals_password: "{{ lookup('password', '/dev/null length=15 chars=ascii_letters,digits') }}"
+    admin_password: "{{ lookup('password', '/dev/null length=15 chars=ascii_letters,digits') }}"
+    cluster_admin_password: "{{ lookup('password', '/dev/null length=15 chars=ascii_letters,digits') }}"
```

Comment:
Not sure why we are setting `cluster_admin_password`. In the overview block we specify an admin password, customer admin password and an evals password?